src/DacpacTool/PackageDeployer.cs

```diff
@@ -182,13 +182,19 @@ private void RunDeploymentScriptFromReferences(FileInfo dacpacPackage, string ta
             }
         }
 
-        private void AddSqlCmdVariables(ScriptExecutionArgs args, string targetDatabaseName)
+        private Dictionary<string, string> AddSqlCmdVariables(string targetDatabaseName)
```

Comment:
Is this necessary? I was assuming this was done because `ScriptExecutionArgs` isn't available now, but we're still instantiating it now?

Reply:
Not sure what you mean - this method is used in the constructor above. ".Variables" is now internal...